tls/test/test_record.py

```diff
@@ -10,9 +10,10 @@
 
 import pytest
 
-from tls.record import (
-    ContentType, ProtocolVersion, TLSCiphertext, TLSCompressed, TLSPlaintext
-)
+from tls._common import enums
+
+from tls.record import (ProtocolVersion, TLSCiphertext, TLSCompressed,
+                        TLSPlaintext)
```

Comment:
Thanks, it is helpful to hear that. Leaving the rest of the `(...)` as is for now!